Date: Thu, 27 Dec 2007 09:40:43 +0900
From: DAIKI MATSUDA <d.matuda@gmail.com>
To: High-Availability Linux Development List <linux-ha-dev@lists.linux-ha.org>
Subject: OCF-resources: SphinxSearchDaemon is not installed

Hi, All.

Congratulations to release Heartbeat 2.1.3.

I read Alan's release mail and made own rpm package. So, I was aware
of difference that SphinxSearchDaemon RA is not installed. And I
attached the simply patch file and please confirm.

Cc: DAIKI MATSUDA <d.matuda@gmail.com>
Signed-off-by: Simon Horman <horms@verge.net.au>

Index: heartbeat/resources/OCF/Makefile.am
===================================================================
--- heartbeat.orig/resources/OCF/Makefile.am	2007-11-30 16:15:19.000000000 +0900
+++ heartbeat/resources/OCF/Makefile.am	2008-01-08 13:16:50.000000000 +0900
@@ -76,6 +76,7 @@ ocf_SCRIPTS	     =  ClusterMon 	\
 			SAPInstance	\
 			SendArp		\
 			ServeRAID	\
+			SphinxSearchDaemon	\
 			Stateful	\
 			SysInfo		\
 			tomcat		\
